AppAuth-JS/built/authorization_service_configuration_test.d.ts

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork